ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.h

```diff
@@ -0,0 +1,50 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+#ifndef BALOVNEV_MODEL_PLUGIN_H
+#define BALOVNEV_MODEL_PLUGIN_H 
+#include <ros/ros.h>
+
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/common/common.hh>
+
+#include <ow_dynamic_terrain/modified_terrain_diff.h>
+
+
+namespace gazebo
+{
+// This plugin is added to a robot description and applies forces calculated 
+// from Balovnev force model
+class BalovnevModelPlugin : public gazebo::ModelPlugin
+{
+public:
+  
```

Comment:
It's good to be explicit about the compiler generated functions.  In this case the default ones are being generated, which looks okay.  But you can be clear that this is what you want with the following (my syntax should be very close if not correct).
```suggestion
  BalovnevModelPlugin() = default;
  BalovnevModelPlugin (const BalovnevModelPlugin&) = default;
  BalovnevModelPlugin& operator= (const BalovnevModelPlugin&) = default;
  ~BalovnevModelPlugin() = default;
```